ui/src/app/dashboard/charts/charts.component.ts

```diff
@@ -39,10 +40,22 @@ export class ChartsComponent implements OnInit {
   };
 
 
-  constructor(private currenciesService: CurrenciesService) {}
+  constructor(
+    private currenciesService: CurrenciesService,
+    private userService: UserService
+  ) {}
 
   ngOnInit() {
     this.currenciesService.getCurrencyCodes().subscribe(v => this.currencies = v.data.currencies);
+    this.userService.findUser('hitler@didnothingwrong.de', ['email', 'firstName']).subscribe(x => console.log(x.data.findClient.email));
```

Comment:
Of course he did nothing wrong

and he didn't know

But it would be nice to remove him

Reply:
-.- boring